pal/audio/native/player.ts

```diff
@@ -132,14 +132,14 @@ export class AudioPlayer implements OperationQueueable {
             }).catch((e) => {});
         }
     }
-    static load (url: string): Promise<AudioPlayer> {
+    static load (url: string, opts?: AudioLoadOptions): Promise<AudioPlayer> {
```

Comment:
Why need opts?

Reply:
because the interface is declare as `AudioPlayer.load (url: string, opts?: AudioLoadOptions)` in `engine/@types/pal/audio.d.ts`  
on framework layer, we pass in opts though we don't need opts on native platform